src/extras/primitives/primitives/a-gltf-model.js

```diff
@@ -0,0 +1,9 @@
+var getMeshMixin = require('../getMeshMixin');
+var registerPrimitive = require('../primitives').registerPrimitive;
+var utils = require('../../../utils/');
+
+registerPrimitive('a-gltf-model', utils.extendDeep({}, getMeshMixin(), {
```

Comment:
Does the material component have an effect on `gltf` mesh? If not, we can remove `getMeshMixin`.